e2e/native-esm/__tests__/native-esm-wasm.test.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// the point here is that it's the node core module
+// eslint-disable-next-line no-restricted-imports
+import {readFileSync} from 'fs';
+// The file was generated by wasm-pack
+import {getAnswer} from '../42.wasm';
```

Comment:
File source: https://github.com/hasharchives/wasm-ts-esm-in-node-jest-and-nextjs

Reply:
Maybe something from https://github.com/mdn/webassembly-examples?

Reply:
that seems reasonable 👍

Reply:
Done in https://github.com/facebook/jest/pull/13505/commits/79e0f40d9397acb923595aa7c9070c24260f4274